### ISSUE TITLE
Execution parameters bug for MaxPayloadinMB

### ISSUE THREAD

**Issue by ericangelokim** [awslabs/mxnet-model-server#854]
https://github.com/awslabs/mxnet-model-server/blob/master/plugins/endpoints/src/main/java/software/amazon/ai/mms/plugins/endpoint/ExecutionParameters.java#L26

'max_request_size' seems to refer to bytes, not mb.

**Comment by vdantu** [comment]
@ericangelokim : True. This was supposed to be an example of how plugins are defined, but its good to correct the example. 

@alexwong
